src/localization/messages_tl.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * Translated default messages for the jQuery validation plugin.
+ * Locale: TL (Tagalog; Tagalog)
+ */
+$.extend( $.validator.messages, {
+	required: "Ang field na ito ay required.",
+	email: "Mangyaring magpasok ng isang valid na email address.",
+	number: "Mangyaring magpasok ng isang valid number.",
+	max: $.validator.format( "Mangyaring magpasok ng value na mas maliit o katumbas ng {0}." ),
```

Comment:
**Full disclosure:** I can't speak Tagalog (nor Filipino).

I improvised this translation because I felt like it wouldn't be ideal to provide the ``min`` translation I had on hand whilst omitting a ``max`` counterpart.